sdk/core/test/helpers/testPlatformTools.ts

```diff
@@ -2,13 +2,17 @@ import { IPlatformEncryptionTools } from '../../src/shared/interfaces'
 
 export const testPlatformTools: IPlatformEncryptionTools = {
   platformName: 'stub',
-  decryptByPrivateKey: async () => {
-    throw new Error('not implemented')
+  // eslint-disable-next-line no-unused-vars
+  decryptByPrivateKey: async (privateKeyBuffer: Buffer, data: string) => {
+    return data
```

Comment:
`decryptByPrivateKey` and `encryptByPublicKey` should be complementary operations.
So we'll need to do `JSON.parse` here.